enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/security/jwt/JWTAuthenticator.java

```diff
@@ -79,6 +79,12 @@ public class JWTAuthenticator implements Authenticator {
     private final JWTValidator jwtValidator = new JWTValidator();
     private final boolean isGatewayTokenCacheEnabled;
     private AbstractAPIMgtGatewayJWTGenerator jwtGenerator;
+    private static String[] whitelistedOrganizationsArray;
+
+    static {
+        whitelistedOrganizationsArray =
+                System.getenv("WHITELISTED_ORGANIZATIONS").split("\\s+");
```

Comment:
This variable name is bit too generic, how about PROD_TOKEN_NONPROD_ALLOWED_ORGS